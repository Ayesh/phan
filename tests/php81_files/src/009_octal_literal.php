<?php
$x = 0o1;
$y = 0O123;
'@phan-debug-var $x, $y';
