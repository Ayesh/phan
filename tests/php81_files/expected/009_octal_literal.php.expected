%s:3 PhanDebugAnnotation @phan-debug-var requested for variable $x - it has union type 1(real=1)
%s:3 PhanDebugAnnotation @phan-debug-var requested for variable $y - it has union type 83(real=83)
